src/redisraft.c

```diff
@@ -1008,6 +1008,100 @@ static int cmdRaftRandom(RedisModuleCtx *ctx,
     return REDISMODULE_OK;
 }
 
+#ifdef HAVE_TLS
+/* Callback for passing a keyfile password stored as a char * to OpenSSL,  copied from redis */
+static int tlsPasswordCallback(char *buf, int size, int rwflag, void *u) {
```

Comment:
just for consistency, I'd write this function with redisraft style.

Reply:
ok, think I did that now.